elife-1234567890-v2/elife-1234567890-v2.xml

```diff
@@ -1973,41 +1991,30 @@ module load anaconda</code>
                     </aff>
                 </contrib>
             </contrib-group>
-            <related-object id="sa0ro1" object-id="10.1101/2021.11.09.467796" object-id-type="id" link-type="continued-by" xlink:href="https://sciety.org/articles/activity/10.1101/2021.11.09.467796"/>
+            <kwd-group kwd-group-type="Importance-of-claim">
+                <!-- Controlled vocabulary of importance values: Landmark; Fundamental; Important; Noteworthy; Useful; Flawed -->
+                <kwd><ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/licenses/by/4.0/">Landmark</ext-link></kwd>
```

Comment:
Why the CCBY link on the `kwd`?

Reply:
A test, to see if it would be valid (it is). Question about it in here: https://docs.google.com/document/d/17xXZGFOKNJHVMyTtPKbKJUkaEZw29qy4zsUudtVA6sM/edit